server/config/main.go

```diff
@@ -43,3 +47,29 @@ func (c *Configuration) IsValid() error {
 
 	return nil
 }
+
+func (c *Configuration) Sanitize() {
+	// Ensure Confluence ends with a slash
+	c.ConfluenceURL = strings.TrimRight(c.ConfluenceURL, "/")
+
+	c.ConfluenceOAuthClientID = strings.TrimSpace(c.ConfluenceOAuthClientID)
+	c.ConfluenceOAuthClientSecret = strings.TrimSpace(c.ConfluenceOAuthClientSecret)
+}
+
+func (c *Configuration) IsOAuthConfigured() bool {
+	return (c.ConfluenceOAuthClientID != "" && c.ConfluenceOAuthClientSecret != "")
+}
+
+func (c *Configuration) ToMap() (map[string]interface{}, error) {
```

Comment:
Where is this being used? In other PRs?

Reply:
line #287 in flow.go and another place also
```
configMap, err := config.ToMap()
	if err != nil {
		return "", nil, nil, err
	}
```